src/controllers/TicketController.java

```diff
@@ -59,10 +60,10 @@ public void updatePriceToDB(int moviename_index, int cinema_index,int showtime_i
 	}
 	
 	
-	public void insertTicketToDB(Movie a, Cinema b, Customer c, String d, String TID, String custName, String custEmail, String custMobileNumber) {
+	public void insertTicketToDB(Movie movie, Cinema cinema, Customer customer, LocalDate showtime, Seat seat, String TID, String custName, String custEmail, String custMobileNumber) {
```

Comment:
Not sure why the changes were not reflected previously? I thought we had changed to standardize the format.